internal/controller/api_to_config.go

```diff
@@ -346,3 +352,15 @@ func sortMapPtr[T any](toSort map[string]*T) []*T {
 	}
 	return res
 }
+
+func joinRawConfigs(raw []v1beta1.RawConfig) string {
+	sort.Slice(raw, func(i, j int) bool {
+		return raw[i].Priority > raw[j].Priority
```

Comment:
this is nondeterministic if `raw` contains multiple with the same prio in a different order everytime (maybe it makes sense to sort the fromK8s cfgs first by name)?